src/main/java/serviceCRUD/editor/UpdateCar.java

```diff
@@ -0,0 +1,79 @@
+package serviceCRUD.editor;
+
+import serviceCRUD.DBWorker;
+import serviceCRUD.validator.InputValue;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+import static serviceCRUD.validator.InputValue.inputItem;
+import static serviceCRUD.validator.InputValue.inputValidateStr;
+
+public class UpdateCar {
+
+    private static final String UPDATE_MODEL = "UPDATE cars SET car_model = ? WHERE id = ?";
+    private static final String UPDATE_NUMBER = "UPDATE cars SET car_number = ? WHERE id = ?";
+    private static final String UPDATE_VIN = "UPDATE cars SET vin = ? WHERE id = ?";
+    private static final String UPDATE_YEAR = "UPDATE cars SET production_year = ? WHERE id = ?";
+    private static final String UPDATE_MILEAGE = "UPDATE cars SET car_mileage = ? WHERE id = ?";
+    private static final String UPDATE_CUSTOMER = "UPDATE cars SET customer = ? WHERE id = ?";
+    private static final String UPDATE_BREAKING = "UPDATE cars SET breaking = ? WHERE id = ?";
+    private static final String UPDATE_SERVICE = "UPDATE cars SET services_id = ? WHERE id = ?";
+
+    PreparedStatement preparedStatement;
+
+    public void getPreparedStatement(DBWorker worker, int id) throws SQLException {
+        String item = String.valueOf(inputItem());
+        switch (item) {
+            case ("MODEL") -> {
```

Comment:
In the real world, we usually update all fields at once, not one by one. But for the learning and practice can be.
Also, enum can be used for field identifiers: MODEL, NUMBER, VIN, etc.